astropy/modeling/rotations.py

```diff
@@ -220,9 +220,9 @@ class EulerAngleRotation(_EulerRotation, Model):
     n_inputs = 2
     n_outputs = 2
 
-    phi = Parameter(default=0, getter=_to_orig_unit, setter=_to_radian)
-    theta = Parameter(default=0, getter=_to_orig_unit, setter=_to_radian)
-    psi = Parameter(default=0, getter=_to_orig_unit, setter=_to_radian)
+    phi = Parameter(default=0, getter=_to_orig_unit, setter=_to_radian, description="Euler angles in deg")
+    theta = Parameter(default=0, getter=_to_orig_unit, setter=_to_radian, description="A 3 character string, a combination of 'x', 'y' and 'z'")
+    psi = Parameter(default=0, getter=_to_orig_unit, setter=_to_radian, description="Euler angles in deg")
```

Comment:
```suggestion
    phi = Parameter(default=0, getter=_to_orig_unit, setter=_to_radian, description="1st Euler angle (Quantity or value in deg)")
    theta = Parameter(default=0, getter=_to_orig_unit, setter=_to_radian, description="2nd Euler angle (Quantity or value in deg)")
    psi = Parameter(default=0, getter=_to_orig_unit, setter=_to_radian, description="3rd Euler angle (Quantity or value in deg)")
```
Note that `Parameter`s can only be numbers, never strings.